packages/kit/src/core/config/index.js

```diff
@@ -29,15 +29,30 @@ export function load_template(cwd, config) {
 
 export async function load_config({ cwd = process.cwd() } = {}) {
 	const config_file = path.join(cwd, 'svelte.config.js');
+	const ts_config_file = path.join(cwd, 'svelte.config.ts');
+	let config;
 
-	if (!fs.existsSync(config_file)) {
-		throw new Error(
-			'You need to create a svelte.config.js file. See https://kit.svelte.dev/docs/configuration'
-		);
+	if (fs.existsSync(config_file)) {
+		config = await import(url.pathToFileURL(config_file).href);
+	} else {
+		if (fs.existsSync(ts_config_file)) {
+			const { loadConfigFromFile } = await import('vite');
+			const resolved_config = await loadConfigFromFile(
+				{
+					command: 'build',
+					mode: process.env.NODE_ENV || 'production'
```

Comment:
the `|| 'production'` shouldn't be needed because it is done here: https://github.com/sveltejs/kit/blob/a8a3b14c4420a135088ef0f5e263fd24e88f0cd6/packages/kit/src/cli.js#L91

i'm worried that this might be putting it into `production` mode when you run `svelte-kit dev`

Reply:
Maybe this command need a `|| 'production'` (?)

https://github.com/sveltejs/kit/blob/a8a3b14c4420a135088ef0f5e263fd24e88f0cd6/packages/kit/src/cli.js#L147-L153
 
Otherwise, I considered using `development` as the default value for `NODE_ENV`.

Reply:
yeah, it seems reasonable to add a `production` default to the `package` command